samples/constitutions/default/actions.js

```diff
@@ -1004,6 +1004,25 @@ const actions = new Map([
       },
     ),
   ],
+  [
```

Comment:
@eddyashton I think we ought to split those actions in their own `virtual_node.js` file, going under `CCF/samples/constitutions/sandbox/` or something similarly distinct from default. `default` should remain a good default choice for a production service, where these actions admittedly don't actively hurt but are also quite unnecessary.

Reply:
I was going to say this would be awkward because we need these actions in most of our tests, where virtual attestations are now applied for all joins. But that's actually wrong - we only need the governance when we want to _change_ the measurements, which should only be for `code_update` and `lts_compat`.

Reply:
A footgun caveat while it's on my mind - the node that `start`s the network always adds its own attestation, so you could still easily accidentally permit virtual nodes if you (somehow) ran a virtual node as the first in the network.

Reply:
True, but this along with the constitution, membership etc is part of what the consortium must review before signing off the transition to open.

If we're worried about accidents, we could decide that running a Start node in virtual mode requires passing --unsafe on the CLI, and will otherwise refuse to start.